mstp.c

```diff
@@ -3252,6 +3275,10 @@ static bool PTSM_run(port_t *prt, bool dry_run)
             }
             if(!(prt->txCount < prt->bridge->Transmit_Hold_Count))
                 return false;
+
+            if(prt->bpduFilterPort)
+                return false;
```

Comment:
as a note, this patch here, differs a bit from the initial one, which was here: https://github.com/mstpd/mstpd/pull/61/commits/83eaaf81d74c954301a1df62429556d44be135d1#diff-afc426ea39d24015ac2bbcb1eed6a712L3256

Reply:
Yeah, seemed prudent to perform the check before the send RSTP/STP check. Can move the check if desired.

Reply:
No, that's fine.
Your patch makes a bit more sense than the initial one.

Reply:
i.e. I understand it easier